### PR TITLE
Support interrupts/exceptions by Qemu log viewer

### PR DESCRIPTION
**Pull request by laerreal**
Now QLV supports Qemu log entries produced by "int" flag of "-d" option.
Some additional entries produced by "in_asm" flag are also supported.

### v2
- fixup non-`TraceInstr` printing in `QLOG_DEBUG` mode
- rename `step` to `interrupt` (`_insert_interrupt_row`)
- fixup coding style, commit messages, typos

### CODE REVIEW

File: qlv.py [hunk 1]
```diff
@@ -230,8 +230,8 @@ def _fill_window(self):
         new_inst_limit = cur_start + TV_WINDOW_SIZE
         new_insts = all_insts[new_inst_idx:new_inst_limit]
 
-        for idx, inst in enumerate(new_insts, new_inst_idx):
```
Comment (BDanAnd) [inline qlv.py:233]: "TraceInstr.icount" in commit message

File: qemu/qlog.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 __all__ = [
     "InInstr",
-    "TraceInstr",
+    "LogStep",
```
Comment (BDanAnd) [inline qemu/qlog.py]: Overlooked this moment. Coding style:
```
__all__ = [
    "InInstr"
  , "LogStep"
      , "TraceInstr"
      , "LogInt"
  , "TBCache"
  , "QTrace"
  , "QEMULog"
]
```

File: qemu/qlog.py [hunk 3]
```diff
@@ -144,6 +165,41 @@ def __str__(self):
...
+class LogInt(LogStep):
+    "It's interrupt/exception handling."
+
+    __slots__ = (
+        "header",
+        "cpu_before",
+        "lineno",
+    )
+
+    # Tt's always "good" unlike `QTrace`. See `QEMULog.trace_stage`.
```
Comment (BDanAnd) [inline qemu/qlog.py]: It's

File: qlv.py [hunk 4]
```diff
@@ -353,6 +363,16 @@ def _insert_step_ignore_diff(self, parent, insert_index, step, tags):
                 values = ("-", "-", str(step))
             )
 
+    def _insert_interrupt_row(self, parent, insert_index, step, tags):
```
Comment (BDanAnd) [inline qlv.py]: Maybe rename `step` to `interrupt` (because `inst` in `_insert_instruction_row`)?

File: qlv.py [hunk 5]
```diff
@@ -37,6 +37,7 @@
 )
 from qemu import (
     TraceInstr,
+    LogInt,
```
Comment (BDanAnd) [inline qlv.py:40]: With `QLOG_DEBUG` == 1:
```
In coroutine "co_trace_builder" (Coroutine based task without description):
Traceback (most recent call last):
  File "common\co_dispatcher.py", line 179, in poll
    ret = next(generator)
  File "qlv.py", line 559, in co_trace_builder
    print("0x%08X: %s" % (i.addr, i.disas))
AttributeError: 'LogInt' object has no attribute 'addr'
```

**Review by BDanAnd — COMMENTED** [review]
v1 viewed

**Comment by laerreal** [comment]
v2 is ready

**Pull request merged**